lib/build.py

```diff
@@ -22,7 +22,7 @@
 try:
   from PyInstaller import main as PyInstallerMain
 except ImportError:
-  logging.error("PyInstaller missing")
+  logging.warn("PyInstaller missing")
```

Comment:
I think we should just make this pass with no message, and let it blow up if someone actually tries to build a client without it.  Add a comment like:

```
# We ignore this failure since most people running the code don't build their own clients and printing an error message causes confusion.  Those building their own clients will need PyInstaller installed.
```